identity-service/src/utils/relayHelpers.js

```diff
@@ -20,7 +20,8 @@ const solanaAudiusAnchorDataProgramId = config.get(
 const allowedProgramIds = new Set([
   solanaClaimableTokenProgramAddress,
   solanaRewardsManagerProgramId,
-  /* secp */ 'KeccakSecp256k11111111111111111111111111111'
+  /* secp */ 'KeccakSecp256k11111111111111111111111111111',
+  /* memo */ 'Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo'
```

Comment:
Ah, we didn't need this for purchases bc we would send tx directly from client, right?